app/models/person.rb

```diff
@@ -155,4 +155,11 @@ def touch_role_appointments
   def truncated_biography
     biography&.split(/\n/)&.first
   end
+
+  def republish_ministerial_pages_to_publishing_api
+    if role_appointments.map(&:ministerial?).any?
```

Comment:
You can golf this a little

```suggestion
    if role_appointments.any?(&:ministerial?)
```

Reply:
Ooh, nice!  Didn't realise this was possible.